openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMMLProvider.java

```diff
@@ -38,7 +38,7 @@ public class LightGBMMLProvider implements TrainingMachineLearningProvider<Light
     /**
      * The reported name of this provider.
      */
-    private static final String PROVIDER_NAME = "Microsoft LightGBM";
+    private static final String PROVIDER_NAME = "Feedzai GBM";
```

Comment:
:warning: !! 

We might have to revert this. Don't know if it's a breaking change. Might require a new major release, as existing models suddenly won't have a jar of their registered provider to use. We can't change this on a hotfix.

And we shouldn't have 2 different provider names.

However, to release this as hotfix, maybe we create a new provider for fgbm and leave lgbm untouched for now.

Reply:
action: Create new provider, and leave LGBM's naming alone. (but Double-check with Saleiro first)